wireframe | component tree

philjacks opened the pull request:
- wireframe image
- component tree image

philjacksZen committed:
wireframe | component tree

Poonam-raj reviewed (changes requested):
We'd ideally want to plan mobile-first which means designing for mobile sized screens first then expanding to desktop size. Would be cool to see the mobile adaptation of this plan.

Would love consideration of what a user posting a comment would look like (where it would lie) and the voting ability would be cool to see visualised too. Filtering is here but what about sorting articles in specific orders? Is that a button/dropdown/other?

Would certain states need to be shared between components aka how will filtering by topic communicate with the articleslist component to do the filtering?

Let me know if your thoughts on this and do another PR with a bit more fleshed out if you can, not much more needed though so good job

philjacksZen committed:
planning updated

philjacks commented:
Yeah good points. I have updated the wireframes & added a mobile view. I've also updated the component tree to show the sort by, upvotes & post comments. I have put notes with each component to show what state I think they would need & where/what requests would be needed.

Poonam-raj reviewed (approved):
Looking good!
